ios/square_reader_sdk.podspec

```diff
@@ -3,7 +3,7 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'square_reader_sdk'
-  s.version          = '3.0.0'
```

Comment:
I am curious why this was `3.0.0` before?

Reply:
Looks like before it was attempting to be synced with this https://github.com/square/reader-sdk-flutter-plugin/blob/7b1cd0aaecd99313e081236d8a12709455bc1982/pubspec.yaml#L3.

I think this podspec is never published to CocoaPods so I think technically we can down version it like this.. but maybe we won't to keep it in sync with the plugin version? So 5.0.0, it looks like is what we're calling this next version.